_posts/2018-08-29-grep-current-word.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: post
+title: 2018 08 29 grep current word
```

Comment:
Can you remove the date digits from the title?

Reply:
Yep, and I nabbed a previous post and fixed it also.